collectors/UcsServerLicenseCollector.py

```diff
@@ -14,13 +14,13 @@ def __init__(self, creds, config):
         }
 
     def describe(self):
-        yield GaugeMetricFamily("ucs_server_license", "licenses")
+        yield GaugeMetricFamily("ucsm_server_license", "licenses")
```

Comment:
@mithunmgore why would it be renamed to `ucsm`? from the name I'd expect `ucs` only.